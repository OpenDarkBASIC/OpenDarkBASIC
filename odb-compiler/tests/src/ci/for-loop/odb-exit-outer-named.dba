outer: for x=1 to 5
    for y=1 to 5
        printc stdout x
        print stdout y
        if x>=3 and y>=3 then exit outer
    next y
next x
