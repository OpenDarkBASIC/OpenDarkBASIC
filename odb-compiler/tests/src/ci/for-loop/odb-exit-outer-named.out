"11\n12\n13\n14\n15\n21\n22\n23\n24\n25\n31\n32\n33\n"
